backend/e4h-services/ingestion-service/app/utils/fieldplan_service_client.py

```
@@ -0,0 +1,104 @@
+
+        payload = {
+            'RequestInfo': request_info.model_dump(by_alias=True, exclude_none=True),
+            'FieldPlanFacility': {
+                'facilityId': facility_id,
+                'fieldPlanId': fieldPlan_id,
+                'isdeleted': False,
+                'tenantId': 'in'
+            }
+        }
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)